installer/templates/phx_assets/daisyui.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * daisyUI Tailwind Plugin. You can update this file by fetching the latest version with:
+ * 
+ * curl -sLo daisyui.js https://cdn.jsdelivr.net/npm/daisyui@5/index.js/+esm
+ * 
+ * Make sure to look at the daisyUI changelog: https://daisyui.com/docs/changelog/
+ * 
+ * =================================================================================
+ * 
+ * Bundled by jsDelivr using Rollup v2.79.2 and Terser v5.37.0.
+ * Original file: /npm/daisyui@5.0.4/index.js
```

Comment:
Hey 💚
daisyUI bundle file is now available for each release on GitHub:  
https://github.com/saadeghi/daisyui/releases/tag/v5.0.8
- Generated with Bun, and has readable JS functions (unlike the jsdelivr file)  
- Can be downloaded from:  
```
curl --sLO https://github.com/saadeghi/daisyui/releases/latest/download/daisyui.js
```

Reply:
Thank you very much! We‘ll adjust it :)